### ISSUE TITLE
Tab glitched out

### ISSUE THREAD

**Issue by Alex-0723** [files-community/Files#10792]
### Description

I had opened files, and attempted to go to Downloads right as my last session opened up. The tab literally phased in and out of existence for a few seconds, then Windows told me that something had gone wrong. Unrelated, but Debugging Files.Package in Visual Studio brought up a bug of referring to something without a present instance, but I don't know the code of this project well enough to understand what that would mean.

### Steps To Reproduce

Basically just do something before the last session is loaded.

### Steps to fix

...Yeah I have no idea. This is a report, nothing more.

### Files Version

2.4.4.0

### Windows Version

Windows 10 22H2 (Not sure)

### Relevant Assets/Logs

I got nothing.

**Comment by Josh-65** [comment]
Can you please upload the debug.log file directly to GitHub (email attachments don't work) . You can get it in Files settings > About > Open log folder.

**Comment by Alex-0723** [comment]
[debug.log](https://github.com/files-community/Files/files/10280057/debug.log)
Also afterwards opening anything in a drive path would just have nothing show up under any folders.

**Comment by yaira2** [comment]
It looks like the installation of Files is messed up, can you try to uninstall and reinstall?

**Comment by Alex-0723** [comment]
I did that, and it turned out I somehow ended up installing the dev version, I think because sideload refused to work for some reason. Anyways it works now, but still nothing shows up. Reinstalling has been tried, but to no avail.
[debug.log](https://github.com/files-community/Files/files/10297514/debug.log)
![Capture](https://user-images.githubusercontent.com/86804957/209409831-fa6d3e5a-29c7-411d-bb48-9311fba16fb5.PNG)
Sorry about the virtual keyboard on the side, that's from EPiKaL Big Bag of Tricks.

**Comment by yaira2** [comment]
Is this an issue in the latest version? v2.4.21

**Comment by Alex-0723** [comment]
The current version I'm using is 2.4.25; I'm still having the same 
issues, and any folders not favorites I can't access. I'll try 
redownloading one last time.

**Comment by yaira2** [comment]
Can you share a screen recording?

**Comment by Alex-0723** [comment]
Everything's good now. 1/24/2023 8:56 PM, yair100 - notifications at 
github.com wrote:
